plugin/storage/kafka/writer.go

```diff
@@ -47,7 +55,8 @@ func NewSpanWriter(producer sarama.AsyncProducer, marshaller Marshaller, topic s
 		}
 	}()
 	go func() {
-		for range producer.Errors() {
+		for e := range producer.Errors() {
+			logger.Error(e.Err.Error())
```

Comment:
Does it make sense to log only once? In your specific example, if the topic is missing, how many times will it be logged?

Reply:
Zap actually supports throttling of logs, so rather than building app-specific logic we could expose full Zap configuration.

Reply:
It seems to log the error I've posted in the description for each submitted span.
Not sure about he throttling but I do see this pretty often:
```json
{"level":"error","ts":1535385948.863829,"caller":"kafka/writer.go:59","msg":"circuit breaker is open","stacktrace":"github.com/jaegertracing/jaeger/plugin/storage/kafka.NewSpanWriter.func2\n\t/go/src/github.com/jaegertracing/jaeger/plugin/storage/kafka/writer.go:59"}
```